sorc/build_gldas.sh

```diff
@@ -8,7 +8,11 @@ USE_PREINST_LIBS=${USE_PREINST_LIBS:-"true"}
 if [ $USE_PREINST_LIBS = true ]; then
   export MOD_PATH=/scratch3/NCEPDEV/nwprod/lib/modulefiles
 else
-  export MOD_PATH=${cwd}/lib/modulefiles
+  if [ $target = jet ]; then
+    export MOD_PATH=/lfs4/HFIP/hfv3gfs/nwprod/hpc-stack/libs/modulefiles/stack
```

Comment:
using hpc-stack here, but not in `module_base.jet`.  Is there a reason for this inconsistency?

Reply:
Can you explain why this if-block is needed or used.

Reply:
@KateFriedman-NOAA Can you check why gldas needs this special treatment and others e.g. gsi, don't?

Reply:
Modification mad as requested